.github/actions/workflow-run-job-linux/action.yml

```diff
@@ -36,12 +36,11 @@ runs:
     - name: Checkout repo
       uses: actions/checkout@v4
       with:
-        path: ${{github.event.repository.name}}
```

Comment:
This clones the repository into `/home/runner/workspace/cccl/cccl/cccl`. I changed it to the default `actions/checkout` behaviour to clone it to `/home/runner/workspace/cccl/cccl`.